content/200-concepts/100-components/01-prisma-schema/06-relations/420-relation-mode.mdx

```diff
@@ -212,25 +212,26 @@ When you apply changes to your Prisma schema with Prisma Migrate or `db push` wi
 
 ### Indexes
 
-One side effect of foreign key constraints in databases is that the database usually also creates an implicit index (for example [MySQL]([https://dev.mysql.com/doc/refman/8.0/en/constraint-foreign-key.html#:~:text=MySQL%20requires%20that%20foreign%20key%20columns%20be%20indexed%3B%20if%20you%20create%20a%20table%20with%20a%20foreign%20key%20constraint%20but%20no%20index%20on%20a%20given%20column%2C%20an%20index%20is%20created.](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html#:~:text=MySQL%20requires%20indexes,as%20described%20previously.))) for the columns you create them for. This is so "foreign key checks can be fast and not require a table scan". 
+In relational databases that use foreign key constraints, the database usually also implicitly creates an index for the foreign key columns. For example, [MySQL will create an index on all foreign key columns](https://dev.mysql.com/doc/refman/8.0/en/create-table-foreign-keys.html). This is to allow foreign key checks to run fast and not require a table scan.
 
-As relation mode `prisma` does not use foreign keys when creating the table, there also no index created and without a manually added index, there is the danger for queries to require full table scans which can be slow or expensive (on database providers that bill per accessed row). To help our users to avoid that, Prisma renders a warning in Prisma Schema when relation mode `prisma` is enabled and a field that is used in an `@relation` does not have an index defined:
+The `prisma` relation mode does not use foreign keys, so no index is created. You instead need to manually add an index on your relation fields with the [`@@index`](/reference/api-reference/prisma-schema-reference#index) attribute (or the [`@unique`](/reference/api-reference/prisma-schema-reference#unique), [`@@unique`](/reference/api-reference/prisma-schema-reference#unique-1) or [`@@id`](/reference/api-reference/prisma-schema-reference#id-1) attributes, if applicable).
```

Comment:
Do we really add an index "on the relation fields"? I think the relation field is `foo @relation...` and the things listed in `fields` we call `relation scalar fields` usually @keerlu